File: DAC/MS.NET/programs/WebApps/BookEFC/Models/Student.cs
using System;
using System.Collections.Generic;

namespace BookEFC.Models;

public partial class Student
{
    public int RollNo { get; set; }

    public string Name { get; set; } = null!;

    public int Age { get; set; }

    public string Email { get; set; } = null!;
}

File: DAC/DSA/LabExam/src/com/cdac/StudentLinkedList/Student.java
package com.cdac.StudentLinkedList;

public class Student {
	public int rollNo;
	public String name;
	public Student next;
	public Student(int rollNo, String name) {
		this.rollNo = rollNo;
		this.name = name;
		this.next = null;
	}
}
